<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTeacher</class>
 <widget class="QDialog" name="DialogTeacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить преподавателя</string>
  </property>
  <widget class="QLabel" name="labr_fio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_fio">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_dolzn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Должность</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_dolzn_min">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_dolzn_min">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дол. (сокр)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_fio_pr">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labr_fio_pr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО (р. п)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_teacher">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_dolzn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_dolzn_rod">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дол. (р. п)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_dolzn_rod">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
